laravel-live-coding/database/migrations/2023_09_18_143750_create_wallets_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wallets', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('currency_id');
+            $table->float('amount')->nullable(false);
+
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->foreign('currency_id')->references('id')->on('currencies');
```

Comment:
it is correct, but in best practice you have to add cascade to keep you DB clean